rohmu/object_storage/base.py

```diff
@@ -121,6 +139,36 @@ def store_file_from_disk(self, key, filepath, metadata=None, multipart=None, cac
     def store_file_object(self, key, fd, *, cache_control=None, metadata=None, mimetype=None, upload_progress_fn=None):
         raise NotImplementedError
 
+    def update_notification_url(self, url: str) -> None:
+        """Allow changing the notificaiton url without having to recreate the object"""
+        self.notification_url = url
+
+    def _queue_notify(self, operation):
+        """Inform notification_url about operation"""
+        if self.notification_url:
+            self.notifier.submit(self._notify, operation)
+
+    def _notify(self, operation):
+        """Actually notify, this gets eventually called from the pool"""
+        try:
+            with get_requests_session() as session:
+                session.post(f"{self.notification_url}", json=operation)
+        except requests.RequestException as ex:
+            self.log.warning("Could not inform robjecter about %r, because %s", operation, ex)
```

Comment:
Does `rohmu` really need to know or mention robjecter explicitly? Shouldn't we consider this a generic webhook implementation?